packages/ibm-products/src/components/FilterSummary/FilterSummary.js

```diff
@@ -20,12 +20,13 @@ let FilterSummary = React.forwardRef(
       clearFiltersText = 'Clear filters',
       clearFilters = () => {},
       filters = [],
+      renderLabel = null,
```

Comment:
Is it necessary to declare it as `null`? If `null` is removed, `renderLabel` will default to `undefined`... same difference.

Or is it a style/standard practice?

Reply:
In general, I like setting a default so I know when I check in the component below I can ensure it's null or undefined. In reality I did this because `renderLabel` is a function but for the check I need it to be null or undefined.